src/main/java/stsjorbsmod/actions/DowngradeCardAction.java

```diff
@@ -0,0 +1,23 @@
+package stsjorbsmod.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.DowngradeableCard;
+import stsjorbsmod.cards.wanderer.Patron;
+
+public class DowngradeCardAction extends AbstractGameAction {
+    private DowngradeableCard card;
+
+    public DowngradeCardAction(DowngradeableCard card) {
+        this.card = card;
+    }
+
+    @Override
+    public void update() {
+        JorbsMod.logger.info("DowngradeCardAction downgrading " + card.toString());
+        card.downgrade();
```

Comment:
It would be nice for this to have an sfx/vfx (reusing an existing one for now would be fine to start). If you do this, same comment as on DestroyCardAction about giving it a duration